tests/calculators/test_qe.py

```diff
@@ -55,20 +68,21 @@ def qe_cmd_and_pseudo(tmp_path_factory):
     pspot_file = tmp_path_factory.getbasetemp() / "Si.UPF"
     shutil_copy(Path(__file__).parent / ".." / "assets" / "QE" / "Si.pz-vbc.UPF", pspot_file)
 
-    return cmd, pspot_file
+    return EspressoProfile(command=pytest_command, pseudo_dir=pspot_file.parent), pspot_file.name
```

Comment:
Once the tests are shifted to rely on ase config, there's no need to return an `EspressoProfile` (it should be constructed automatically from the ase config file), we only need to specify the `pspot_file.name`. I would move the setting up of the pseudo dir to the github pytest setup and only have this function moving the `Si.UPF` to the `pseudo_dir` location, specified by the `espresso` in ase config file. (Then the `kw` would not need a `profile` at all in the tests below)

Reply:
I wanted to test both automatically reading the profile from the config file and explicitly passing it to the wrapper.

Reply:
The thing that makes it all tricky is that the tests need to use their own pseudopotentials, but the profile includes information on where the pseudopotential directory is.  So you _can't_ just rely on the user's generic profile (unless you copy the testing PPs into their directory).  I'll try to document this in the files a bit more carefully